docs/source/wrapper_development.md

```diff
@@ -96,6 +96,94 @@ This allows us to group and clearly define which components should and which one
  - [Syntactic layer](./detailed_design.md#syntactic-layer): 
    If none is available, one must be developed.
 
+To facilitate the creation of the session class on the interoperability layer, 
```

Comment:
Perhaps it makes sense to repeat and put into context the short explanation of the one in the user's guide:
> When you add an object to a wrapper, a copy of the object is created in the registry belonging to the session of the wrapper.

Perhaps in the developer's docs more information can be given.

Reply:
I agree, but I think this is not related to this issue/pull request. Such a change would actually mean doing a revision of the wrapper development tutorial.

Reply:
Can you then create the new issue?

Reply:
Perhaps for now we can add just a reference to it.

Reply:
Issue #124 created.